Assignment4/pablo-crucera-21A314/task_06.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""Task_06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1JrYMnDg5eLyRgZZg6KMc9pbmBzkHnc1n
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This will not run in Python, you need to comment out this line. The same applies for other files